### PR TITLE
Translate blocks into English using the drop-down

### PR DESCRIPTION
**Pull request by OregSamSas**
This PR is to solve an issue emerged when trying to translate blocks into English (for example using the drop-down found above the text box containing the translated scratchblocks on the translator page (that is reached by https://github.com/scratchblocks/scratchblocks.github.io/pull/10)).

(Needed for implementing the suggestion in #491)

(Little piece of history:
Issue: only "en" was shown in the drop-down, and there was no `scratchblocks.allLanguages.en.name` set. => ~~tried to solve it, by adding an additional line, which adds a `name` property to it. ("English")~~ (It was a mistake)
Later it was found out, as @tjvr gently pointed out, the name property should be set (already) in the built-in locale for English.)

### CODE REVIEW

File: locales-src/build-locales.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import extraAliases from "./extra_aliases.js"
 import scratch_l10n from "scratch-l10n"
 // We can't `import {default}` since it's a reserved word.
 const localeNames = scratch_l10n.default
+localeNames.en.name = "English";
```
Comment (tjvr) [inline locales-src/build-locales.js:10]: I suspect you'll want to update the built-in English locale here: https://github.com/scratchblocks/scratchblocks/blob/ca161ab3bd718d1e602f5c49faf2b1c8fdac25fd/syntax/blocks.js#L194

File: index.html [hunk 2]
```diff
@@ -94,7 +94,6 @@ <h1>
 var languageCodes = Object.keys(scratchblocks.allLanguages)
 languageCodes.sort()
 languageCodes.forEach(function(code) {
-  if (code === "en") return
```
Comment (tjvr) [inline index.html:97]: You'll want to make this change in the other repository: https://github.com/scratchblocks/scratchblocks.github.io/

This repo is the library, that one is the homepage. :)

Comment (OregSamSas) [reply index.html:97]: Thank you very much.

File: locales-src/build-locales.js (hunk 1 above)
Comment (OregSamSas) [reply locales-src/build-locales.js]: Yes, that was right.

**Comment by OregSamSas** [comment]
I've created a PR in the repo for the page scratchblocks.github.io. (https://github.com/scratchblocks/scratchblocks.github.io/pull/10) I only removed the disuse of English in the translator's drop-down.

**Review by tjvr — APPROVED** [review]

**Pull request merged**